tools/lint/config/eslint.config.js

```diff
@@ -1,14 +1,14 @@
 module.exports = {
   parser: '@babel/eslint-parser',
-  plugins: ['@emotion', 'simple-import-sort'],
+  plugins: ['@emotion', '@lg-tools', 'simple-import-sort'],
   extends: [
     'eslint:recommended',
-    'plugin:react/recommended',
-    'plugin:react-hooks/recommended',
     'plugin:jest/recommended',
-    'prettier',
     'plugin:jsx-a11y/recommended',
+    'plugin:react/recommended',
+    'plugin:react-hooks/recommended',
     'plugin:storybook/recommended',
+    'prettier',
```

Comment:
should we add a basic rule inside of our plugin that we can turn on in this PR?

Reply:
Not that wouldn't result in a bunch of lint errors...

Reply:
I'd have to write a rule that we 100% follow right now

Reply:
Enabled `"no-indirect-imports"`, which we _have_ to follow otherwise rollup won't build